libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -857,3 +857,16 @@ bool AP_MotorsMulticopter::arming_checks(size_t buflen, char *buffer) const
 
     return true;
 }
+
+#if APM_BUILD_TYPE(APM_BUILD_UNKNOWN)
```

Comment:
Should we have the #if in the .h file too?
I am assuming the answer is no but better to ask a stupid question...

Reply:
It would be nice but the current build system doesn't allow it, header files cant "see" the `APM_BUILD_TYPE` define.